packages/api/src/routes/mint.ts

```diff
@@ -86,13 +91,140 @@ const mint: FastifyPluginAsync = async (fastify, opts): Promise<void> => {
       }
 
       // Build message to sign
-      const encodedPlayerMedals = encodeMedals(player.medals)
-      const encodedRanchMedals = encodeMedals(player.medals)
-      const message = web3.utils.encodePacked(
-        request.body.address,
-        player.points,
-        encodedPlayerMedals,
-        encodedRanchMedals
+      // This is the solidity function we will need to call:
```

Comment:
IMHO, we can remove this comment before merge or create an array literal type for the arguments of the `mintFarmerAwards` function

Reply:
The types should be the same as the `MintOutput.data` types.